datashuttle/tui/app.py

```diff
@@ -0,0 +1,72 @@
+from pathlib import Path
+from time import monotonic
+
+from textual.app import App, ComposeResult
+from textual.binding import Binding
+from textual.widgets import (
+    DirectoryTree,
+    Footer,
+    Header,
+    Input,
+    Label,
+    TabbedContent,
+    TabPane,
+)
+
+
+class TuiApp(App):
+    """
+    The main app page for the DataShuttle TUI.
+
+    Running this application in a main block as below
+    if __name__ == __main__:
+         app = MyApp()
+         app.run()
+
+    Initialises the TUI event loop and starts the application.
+    """
+
+    TITLE = "DataShuttle"
+
+    # Set path to TCSS
```

Comment:
I would suggest removing this comment as from the nice variable names it is clear what is going on